## Welcome to GitHub Pages

You can use the [editor on GitHub](https://github.com/kosticlab/microbialgenes.github.io/edit/gh-pages/index.md) to maintain and preview the content for your website in Markdown files.

Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# The pan-ecological gene universe of metagenomics

This resource accompanies a submitted manuscript exploring the gene content of microbiomes (microbial communities) across 17 different ecologies, including humans, animals, and other environmental sources. The link below will lead you to a download page for our gene resources, and the abstract describes our findings, which notably include an unprecendent appreciation for conservation of certain genes across different environments.

When our work is available online, we will include a link to the manuscript here.

## Abstract 

The human microbiome consists of microbes with pan-ecological evolutionary origins, yet a systematic gene-level analysis of microbial life across ecologies is lacking. We quantified the gene content from 14,183 samples across 17 ecologies -- 6 human-associated, 7 non-human-host associated (e.g. mouse gut), and 4 in other environmental niches (e.g. soil). At 30% amino acid identity, we identified 117,629,181 non-redundant genes across all samples, 66% of which were singletons, only being observed in one sample. We quantified the genetic similarity and “uniqueness” between different ecologies, showing that sites like the human vaginal and skin ecologies had low genetic alpha-diversity yet high beta-diversity, indicating few species but high pangenomic variation. We further identified a set of 1,864 sequences conserved across all ecologies, which indicates an overwhelming gene-level conservation to microbial life despite extreme taxonomic variation. However, using 90% amino acid clustering identity, we did not observe any globally conserved genes, even those known to be present in all bacteria. This indicates that prior studies, which cluster at, for example, 95% nucleotide identity, have not estimated microbial gene content accurately. We additionally found genes that were differentially abundant in particular groups of ecologies (e.g. human gut and non-human gut genes), identifying discrete functions among these groups. We showed that genes associated with pathogenic taxa tend to be the most likely to appear in multiple ecologies. We provide our databases, as well as the sets of genes described above, as a resource at https://microbial-genes.bio/. 

## Resource

You can access our gene catalog resources (FASTA files, functional gene annotations, etc) at our <a href="https://figshare.com/projects/A_pan-ecological_metagenomic_database_reveals_over_1_800_genes_conserved_across_microbial_ecosystems/112590"> figshare resource.</a> 

[Link](url) and ![Image](src)
```

For more details see [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).

### Jekyll Themes

Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/kosticlab/microbialgenes.github.io/settings/pages). The name of this theme is saved in the Jekyll `_config.yml` configuration file.

### Support or Contact

Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.
